pandas/core/window/rolling.py

```diff
@@ -1880,18 +1925,18 @@ def validate(self):
         # we allow rolling on a datetimelike index
         if (
             self.obj.empty
+            # TODO: add "or self.is_datetimelike"?
             or isinstance(
                 self._on, (ABCDatetimeIndex, ABCTimedeltaIndex, ABCPeriodIndex)
             )
         ) and isinstance(self.window, (str, BaseOffset, timedelta)):
 
             self._validate_monotonic()
 
-            # we don't allow center
-            if self.center:
+            # we don't allow center for offset based windows
```

Comment:
This entire clause should be removed once `center` is supported

Reply:
removed this now, too! :)